### PR TITLE
Papa 10a gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Deployment times and locations can be reviewed in Papa 10a QLR (3203-01005)
Sensor configurations can be reviewed in FCPs (3304-00022-00078 and 3304-00022-00079)

File: deployment/GP05MOAS-G0899_Deploy.csv [hunk 1]
```diff
@@ -2,4 +2,8 @@ CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentN
...
-SKQ202308S,,SKQ202308S,,GP05MOAS-G0899-04-CTDGVM000,1,1,2023-05-16T15:46:00,2023-05-18T03:20:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-CTDGVM-09686,50.47069,-144.83979,,1000,1000,,,
\ No newline at end of file
+SKQ202308S,,SKQ202308S,,GP05MOAS-G0899-04-CTDGVM000,1,1,2023-05-16T15:46:00,2023-05-18T03:20:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-CTDGVM-09686,50.47069,-144.83979,,1000,1000,,,
+SU20231023,,SU20231023,,GP05MOAS-G0899-00-ENG000000,2,1,2023-10-24T03:55:00,2023-10-24T07:17:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGCON-GLDROO-00899,44.421,-125.274,,1000,1000,,,
+SU20231023,,SU20231023,,GP05MOAS-G0899-01-FLORDM000,2,1,2023-10-24T03:55:00,2023-10-24T07:17:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-FLORDM-06990,44.421,-125.274,,1000,1000,,,
+SU20231023,,SU20231023,,GP05MOAS-G0899-02-DOSTAM000,2,1,2023-10-24T03:55:00,2023-10-24T07:17:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-DOSTAM-00323,44.421,-125.274,,1000,1000,top mount,,
+SU20231023,,SU20231023,,GP05MOAS-G0899-04-CTDGVM000,2,1,2023-10-24T03:55:00,2023-10-24T07:17:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-CTDGVM-09686,44.421,-125.274,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-G0899_Deploy.csv:9]: looks good

File: deployment/GP05MOAS-GL276_Deploy.csv [hunk 2]
```diff
@@ -10,4 +10,8 @@ SR1811,,TY-1902,,GP05MOAS-GL276-04-CTDGVM000,2,1,2018-07-23T06:33:00,2019-02-12T
...
-SU20211112,,SU20211112,,GP05MOAS-GL276-04-CTDGVM000,3,1,2021-11-13T09:25:00,2021-11-13T13:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-09238,44.78732,-125.31298,,1000,1000,short deployment due to steering issues,,
\ No newline at end of file
+SU20211112,,SU20211112,,GP05MOAS-GL276-04-CTDGVM000,3,1,2021-11-13T09:25:00,2021-11-13T13:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-09238,44.78732,-125.31298,,1000,1000,short deployment due to steering issues,,
+SU20231023,,,,GP05MOAS-GL276-00-ENG000000,4,1,2023-10-24T03:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGCON-GLDROO-00276,44.4206,-125.2718,,1000,1000,,,
+SU20231023,,,,GP05MOAS-GL276-01-FLORDM000,4,1,2023-10-24T03:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORDM-03583,44.4206,-125.2718,,1000,1000,,,
+SU20231023,,,,GP05MOAS-GL276-02-DOSTAM000,4,1,2023-10-24T03:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00398,44.4206,-125.2718,,1000,1000,top mount,,
+SU20231023,,,,GP05MOAS-GL276-04-CTDGVM000,4,1,2023-10-24T03:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-09238,44.4206,-125.2718,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-GL276_Deploy.csv:17]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Review by cooleyky — APPROVED** [review]
Looks good

**Pull request merged**